package.json

```diff
@@ -2,7 +2,9 @@
   "name": "melange-jest",
   "license": "MIT",
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.10.0",
-    "jest": "^26.5.2"
+    "@testing-library/jest-dom": "^6.4.1",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
+    "jsdom": "^24.0.0"
```

Comment:
I don't think this is needed because `jest-environment-jsdom` already depends on it: https://www.npmjs.com/package/jest-environment-jsdom?activeTab=dependencies.

However, we could update the opam `depext` field to reflect this new dependency.